docs/src/main/asciidoc/neo4j.adoc

```diff
@@ -429,7 +429,7 @@ Please add the following dependency management and dependency to your `pom.xml`
       <dependency>
           <groupId>io.projectreactor</groupId>
           <artifactId>reactor-bom</artifactId>
-          <version>Californium-SR4</version>
+          <version>Californium-RELEASE</version>
```

Comment:
I'm uncomfortable not using the same versions in documentation and ITs.

Considering the extension is experimental, we could point to the new Neo4j version in the doc too?

Reply:
And rightfully uncomfortable. That was an editing mistake on my side. I fixed this.